support/COMMUNITY_CONTACTS.md

```diff
@@ -68,19 +68,19 @@ rotation, feel free to send a PR to remove yourself.
 
 | week       | contact                                             |
 | ---------- | --------------------------------------------------- |
-| 2020-09-14 | [julz](https://github.com/julz)                     |
-| 2020-09-21 | [JRBANCEL](https://github.com/JRBANCEL)             |
-| 2020-09-28 | [nak3](https://github.com/nak3)                     |
-| 2020-10-05 | [vagababov](https://github.com/vagababov)           |
-| 2020-10-12 | [tcnghia](https://github.com/tcnghia)               |
-| 2020-10-19 | [dprotaso](https://github.com/dprotaso)             |
-| 2020-10-26 | [mattmoor](https://github.com/mattmoor)             |
-| 2020-11-02 | [ZhiminXiang](https://github.com/ZhiminXiang)       |
-| 2020-11-09 | [yanweiguo](https://github.com/yanweiguo)           |
-| 2020-11-16 | [markusthoemmes](https://github.com/markusthoemmes) |
-| 2020-11-23 | [julz](https://github.com/julz)                     |
-| 2020-11-30 | [JRBANCEL](https://github.com/JRBANCEL)             |
-| 2020-12-07 | [nak3](https://github.com/nak3)                     |
 | 2020-12-14 | [tcnghia](https://github.com/tcnghia)               |
 | 2020-12-21 | (looking for a volunteer)                           |
 | 2020-12-28 | [vagababov](https://github.com/vagababov)           |
+| 2021-01-04 | [mattmoor](https://github.com/mattmoor)             |
```

Comment:
Ack.  I will put this on my calendar and follow up with folks as they come back.

Reply:
👍